flocker/provision/_install.py

```diff
@@ -800,6 +802,57 @@ def task_open_control_firewall(distribution):
     Open the firewall for flocker-control.
     """
     if is_centos(distribution):
+        open_firewall = open_firewalld
+    elif distribution == 'ubuntu-14.04':
+        open_firewall = open_ufw
+    else:
+        raise DistributionNotSupported(distribution=distribution)
+    return sequence([
+        open_firewall(service)
+        for service in ['flocker-control-api', 'flocker-control-agent']
+    ])
+
+
+def catch_exit_code(expected_exit_code):
+    """
```

Comment:
:grey_question: I was going to ask for a sentence here describing what the function does, but reading the rest of the docstring it seems redundant. No action required.